iso/boot/grub/grub.cfg

```diff
@@ -0,0 +1,7 @@
+set timeout=1
+set default=0
```

Comment:
`timeout` -> This is a countdown before boot the operating system, in that case, the GRUB menu will be shown for a second before boot up the OS.

`default` -> The OS to be booted up by default, in that case, it will be the entry 0, because in the grub,cfg we can have many entries as possible, so the default entry will be 0 -> First